### ISSUE TITLE
Initialized parameters

### ISSUE THREAD

**Issue by seanzhuh** [ubc-vision/RefTR#3]
Hi, a great job,

Do you use ImageNet pretrained parameters to initialize the model or the parameters of pre-trained DETR on MS-COCO dataset ? In configs/refcoco+/RefTR_refcoco+.sh, at line 12, --pretrained_model "./data/MODEL_ZOO/detr-r50-e632da11.pth", but in paper, you claim "For image backbone, we experiment with the popular ResNet-50 and ResNet-101 networks [15] where weights are initialized from corresponding ImageNet-pretrained models", it's confused. 

By the way, how to train RefTR separately instead of joint multi-task training ?

Thanks.

**Comment by jojo23333** [comment]
Hi, thanks for your attention. Your question is a good catch, at first we use detr~r50 pretrained weights on mscoco, then i realized mscoco training could have some slightly overlap with refcoco+ validation, so switched back to imagenet init.(to switch back, you simply remove that line.) As what I experienced, This leads to some performances drop. To way to makeup for this is simple, a detr can be retrained with overlap data removed from mscoco. Sorry for the inconsistent here, some experiments were redone after the submission of the paper. So the conclusion is transfer learning from detection pretrained model is important, given refcoco's small size is problematic for training a transformer. But you have to carefully handle mscoco and refcoco given they have some overlaps.

**Comment by jojo23333** [comment]
To train seperately, you can just disable one of the detection head or Segmentation head(remove --mask).

**Comment by sean-zhuh** [comment]
Thanks a lot, I'm just wondering what is the performance of the single task model.

**Comment by seanzhuh** [comment]
Thanks a lot to clarify!
